FuelLabs/fuel-indexer
# fix: update `forc index`'s `--native` indexer creation and usage

### Description

- Whilst initially looking into https://github.com/FuelLabs/fuel-indexer/issues/966 wanted to use `chrono` to keep track of block indexing time, but can't use it in WASM execution cause of https://github.com/FuelLabs/fuel-indexer/issues/408, so decided to use native. 
- Noticed that some aspects of native don't work via the plugin
- Just now noticing this cause no one (rightfully so, at this time) uses native execution

### Testing steps

- [ ] Build `forc-index` locally `cargo build -p forc-index --release`
  - Use this ☝🏽 forc-index when testing
- [ ] Create a native indexer `forc-index new indexer-test --namespace fuel --native && cd indexer-test`
- [ ] Build it `forc-index build --native`
- [ ] Run it `cargo run -- --fuel-node-host beta-3.fuel.network --fuel-node-port 80 --run-migrations --manifest indexer_test.manifest.yaml`
  - Yes running via `cargo` is not ideal and not the final state
  - Will update this later (again, no one uses this as of now, but it should still work decently) 

### Changelog

- fix: update `forc index`'s `--native` indexer creation and usage
